wazuhapp/opensearch-dashboards/Docker/build.sh

```diff
@@ -46,12 +47,16 @@ change_node_version () {
 
 
 prepare_env() {
-    echo "Downloading package.json from wazuh-kibana-app repository"
+    echo "Downloading package.json and .nvmrc from wazuh-kibana-app repository"
     if ! curl $wazuh_app_package_json_url -o "/tmp/package.json" ; then
         echo "Error downloading package.json from GitHub."
         exit 1
     fi
 
+    if ! curl $wazuh_app_nvmrc_url -o "/tmp/.nvmrc" ; then
+        echo "Error downloading package.json from GitHub."
```

Comment:
**nitpick**: replace `package.json` by `.nvmrc`

Reply:
Solved in [b8ffbce](https://github.com/wazuh/wazuh-packages/pull/2395/commits/b8ffbcea43a36bf146a5bad907a622f3c422238b)